app/assets/schemas/archive-format.json

```diff
@@ -0,0 +1,203 @@
+              "type": "string"
+            }
+          }
+        },
+        "contacts": {
+          "id": "https://the-federation.info/specs/backup-restore#archive_format/user/contacts",
+          "type": "array",
+          "items": {
+            "id": "https://the-federation.info/specs/backup-restore#archive_format/user/contacts/185",
+            "type": "object",
+            "properties": {
+              "sharing": {
+                "id": "https://the-federation.info/specs/backup-restore#archive_format/user/contacts/185/sharing",
+                "type": "boolean"
+              },
+              "following": {
+                "id": "https://the-federation.info/specs/backup-restore#archive_format/user/contacts/185/receiving",
+                "type": "boolean"
+              },
+              "person_thefederation_uid": {
+                "id": "https://the-federation.info/specs/backup-restore#archive_format/user/contacts/185/person_thefederation_uid",
+                "type": "string"
+              },
+              "contact_groups": {
+                "id": "https://the-federation.info/specs/backup-restore#archive_format/user/contacts/185/contact_groups",
+                "type": "array",
+                "items": {
+                  "id": "https://the-federation.info/specs/backup-restore#archive_format/user/contacts/185/contact_groups/0",
+                  "type": "object",
+                  "properties": {
+                    "name": {
+                      "id": "https://the-federation.info/specs/backup-restore#archive_format/user/contacts/185/contact_groups/0/name",
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "name"
+                  ]
+                }
+              }
+            },
+            "required": [
+              "sharing",
+              "following",
+              "person_thefederation_uid"
+            ]
+          }
+        },
+        "posts": {
+          "id": "https://the-federation.info/specs/backup-restore#archive_format/user/posts",
+          "type": "array",
+          "items": {
+            "id": "https://the-federation.info/specs/backup-restore#archive_format/user/posts/131",
+            "type": "object",
+            "properties": {
+              "guid": {
+                "id": "https://the-federation.info/specs/backup-restore#archive_format/user/posts/131/guid",
+                "type": "string"
+              },
+              "text": {
+                "id": "https://the-federation.info/specs/backup-restore#archive_format/user/posts/131/text",
+                "type": "string"
+              },
+              "public": {
+                "id": "https://the-federation.info/specs/backup-restore#archive_format/user/posts/131/public",
+                "type": "boolean"
+              },
+              "thefederation_uid": {
```

Comment:
Maybe not for discussion here, but I find that name rather weird. I'd prefer something more neutral like `accountIdentifier` or similar. What do you think?